src/components/KeyPressContext.js

```diff
@@ -44,7 +44,15 @@ export function useKeysPressed(allowedKeys) {
 	return pressedKeys;
 }
 
-const isDarwin = /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
+let onMac = null;
+
+if (typeof window === 'undefined') {
```

Comment:
in JS `undefined` is a value and not a string. 

![image](https://user-images.githubusercontent.com/2099301/144278117-c27452fc-7956-41e7-a5c8-61e338f2de40.png)

Reply:
So I looked into this, and you are correct that undefined is a value not a string... But SURPRISE! typeof returns undefined as a string... which is totally wierd, but here we are.

```
[] :~/projects/wm-cli mlakewood$ node
Welcome to Node.js v16.4.2.
Type ".help" for more information.
> typeof window
'undefined'
> typeof window === 'undefined' ? 1 : 0
1
> typeof window === 'undefined' ? "its a string" : "its something else"
'its a string'
```

So I think this code is correct.

Reply:
you're right, I misread the check. It does do what it says